FREYA-1757: Add 'verbose_name' to all apps

i-oden opened the pull request:
### 📋 Summary
At the moment only the 'articles' app has the attribute 'verbose_name' defined in the AppConfig. This PR adds that attribute to the remaining apps.
 
### 🛠️ Changes Made
- `verbose_name` attribute added to the AppConfig's in the remaining apps
- docstrings added to the AppConfig's where they are missing, and in a couple cases adds a 'TODO' for adding more information 

### 🔍 Notes for Reviewers
The docstrings aren't a must, just added them for consistency.
 
### ✅ Checklist
- [x] PR title follows the pattern: `FREYA-XXXX: Clear and short description`
- [x] Jira / Github issue is linked in description
- [x] Assignee is selected
- [x] Code and content adhere to [conventions](https://scilifelab.atlassian.net/wiki/spaces/CDP2/pages/1909424133/Guidelines+for+the+portal+content)
- [x] Automated checks pass
- [x] Reviewer is selected when the PR is marked as ready for review

### 🔗 Jira Issue
Closes: [FREYA-1757](https://scilifelab.atlassian.net/browse/FREYA-1757)

senthil10 reviewed (commented):
Thank for doing this, noticed some files missing the empty newline at the end :)

And I have commented a shot description for `news`, for `data management` and `privacy`, maybe @LianeHughes can help.

LianeHughes commented on pages/news/apps.py:
```diff
@@ -2,5 +2,10 @@
 class NewsConfig(AppConfig):
+    """Configuration for the news app.
+    
+    TODO: Add short description of news app here.
```
(and let me know if you need any more writing)

i-oden replied:
Thanks! I'll add these to the docstrings and rerequest review.

i-oden commented:
> Looks good to me. just need newlines. Also there are unnecessary white spaces in the endings of docstrings, as well as missing empty lines between functions and file docstring. I would recommend to run either:
> 
> ```
> ruff format
> ```
> 
> or
> 
> ```
> docker compose exec web ruff format
> ```
> 
> as we are running in the docker.
> 
> It will automatically resolve formatting issues and make files compliant with ruff checks.

Regarding this -- when I run ruff now (thanks for that) it says that 18 files will be changed. It's the same version as your github action is running (0.14.5), but that means that the checks should actually be failing in the PRs and not pass as they are doing now. Should I add this as a todo for me, we could merge this first and then I could make sure the action catches formatting issues and fix the ones that are incorrect atm incl the empty lines etc?  @ScilifelabDataCentre/teamfreya

senthil10 commented on pages/data_management/app.py:
I generally advocate to open new PR if the changes made are not relevant to the main goal of current PR.

But in this case, the current PR changes and changes that would made in new PR are very small, so I think we can just make these changes (filename fix and single quotes fix) in this PR if it is ok with you.

i-oden commented:
> > Looks good to me. just need newlines. Also there are unnecessary white spaces in the endings of docstrings, as well as missing empty lines between functions and file docstring. I would recommend to run either:
> > ```
> > ruff format
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > or
> > ```
> > docker compose exec web ruff format
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > as we are running in the docker.
> > It will automatically resolve formatting issues and make files compliant with ruff checks.
> 
> Regarding this -- when I run ruff now (thanks for that) it says that 18 files will be changed. It's the same version as your github action is running (0.14.5), but that means that the checks should actually be failing in the PRs and not pass as they are doing now. Should I add this as a todo for me, we could merge this first and then I could make sure the action catches formatting issues and fix the ones that are incorrect atm incl the empty lines etc? @ScilifelabDataCentre/teamfreya

Ok I've had a look now and the action does pass because of linting. But imo we should add a format check here too. Ok if I add that in another PR so that we catch formatting issues as well?

senthil10 reviewed (dismissed):
This PR's main objective to add `verbose_name` to all the apps, so it looks good to me.

As you said, the observed format issue/file name issue can be fixed in new PR which would focus on fixing the format issues on the whole repo 👍

i-oden commented:
Waiting for https://github.com/ScilifelabDataCentre/swedish-pathogens-portal/pull/40

i-oden commented on pages/news/apps.py:
```diff
@@ -2,5 +2,11 @@
 class NewsConfig(AppConfig):
+    """Configuration for the news app.
+
+    TODO: Add short description of news app here.
```
Yeah I missed this one, not intentional.

i-oden replied:
Fixed ✅

i-oden commented:
> Great work. Not sure whether it is intentional of just missed by chance. this [file](https://github.com/ScilifelabDataCentre/swedish-pathogens-portal/blob/FREYA-1757-add-verbose-name-to-existing-apps/pages/data_management/apps.py) missing verbose name and docstring.

Yeah it was the conflict resolution, because I changed the name of the file. Missed that. Fixed now.
